src/policy/fees.cpp

```diff
@@ -525,6 +610,7 @@ void CBlockPolicyEstimator::processBlock(unsigned int nBlockHeight,
 
     LOG(ESTIMATEFEE, "Blockpolicy after updating estimates for %u confirmed entries, new mempool map size %u\n",
         entries.size(), mapMemPoolTxs.size());
+    feeStats.UpdateSizeDeque(nBlockSize);
```

Comment:
Similarly to what @ptschip wrote above, I wonder whether it is a good idea to expose `deque` as an implementation detail here. Can't that be hidden within `feeStats`?